Lib/test/test_ssl.py

```diff
@@ -1375,11 +1375,29 @@ def test_set_ecdh_curve(self):
         ctx = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
         ctx.set_ecdh_curve("prime256v1")
         ctx.set_ecdh_curve(b"prime256v1")
+        # Only OpenSSL 3 and above supported for multiple curves
+        if (IS_OPENSSL_3_0_0 >= 3):
+            ctx.set_ecdh_curve("prime256v1:brainpoolP384r1")
+            ctx.set_ecdh_curve(b"prime256v1:brainpoolP384r1")
         self.assertRaises(TypeError, ctx.set_ecdh_curve)
         self.assertRaises(TypeError, ctx.set_ecdh_curve, None)
         self.assertRaises(ValueError, ctx.set_ecdh_curve, "foo")
         self.assertRaises(ValueError, ctx.set_ecdh_curve, b"foo")
-
+        # Multiple bad curves should cause error for any OpenSSL version
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, "foo:bar")
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, b"foo:bar")
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, "prime256v1:bar")
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, b"prime256v1:bar")
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, "foo:prime256v1")
+        self.assertRaises(ValueError, ctx.set_ecdh_curve, b"foo:prime256v1")
+        #self.assertRaises(ValueError, ctx.set_ecdh_curve, ":")
```

Comment:
what's up with the commented out test cases?  are these not valuable?  it seems like a set of edge cases worth covering to define behavior on unusual inputs.